src/components/Button/Button.tsx

```diff
@@ -0,0 +1,64 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+
+import { BaseButtonProps } from "./BaseButton";
+
+import "lbh-frontend/lbh/components/lbh-button/_button.scss";
+
+/**
+ * The proptypes for {@link Button }
+ */
+export interface ButtonProps extends BaseButtonProps {
```

Comment:
I'm not sure I like `BaseButtonProps` as a concept. It makes it hard to find the props for this component. I think this code would be easier to read if we duplicated those properties in each component's props, especially since there's only one non-standard prop: `isStartButton`.